compiler/tools/fpp-to-dict/test/dictionary.schema.json

```diff
@@ -261,6 +261,35 @@
                             "members",
                             "default"
                         ]
+                    },
+                    {
+                        "type": "object",
+                        "properties": {
+                            "kind": {
+                                "type": "string",
+                                "enum": [
+                                    "alias"
+                                ]
+                            },
+                            "qualifiedName": {
+                                "type": "string"
+                            },
+                            "type": {
+                                "$ref": "#/$defs/typeDescriptor"
```

Comment:
@jwest115 Can you explain what is going on in line 278? It doesn't look right. Why are there dollar signs, a number sign, and slashes?

Reply:
It's part of JSON Schema's syntax for referencing an internal sub schema (in this case, the typeDescriptor schema specified in line 8). More info at https://json-schema.org/understanding-json-schema/structuring#defs.

Reply:
Thanks! I didn't realize this was part of the schema. It looked like garbled JSON.